<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>550</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Fixedsys</family>
    <pointsize>9</pointsize>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>PDF Maker</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/file.ico</normaloff>:/icons/file.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
	font: 9pt &quot;Fixedsys&quot;;
}

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="topbar" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>35</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #403E3E;
color : #FFFFFF;</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="title">
         <property name="maximumSize">
          <size>
           <width>178</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Fixedsys</family>
           <pointsize>14</pointsize>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 14pt &quot;Fixedsys&quot;;</string>
         </property>
         <property name="text">
          <string>PDF Editor</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="navbar" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #403E3E;
color : #FFFFFF;</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="loadButton">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	font: 9pt &quot;Fixedsys&quot;;
	border: none;
}
QPushButton:hover{
	color:#D48A21;
}</string>
         </property>
         <property name="text">
          <string>Load text</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveButton">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	font: 9pt &quot;Fixedsys&quot;;
	border: none;
}
QPushButton:hover{
	color:#D48A21;
}</string>
         </property>
         <property name="text">
          <string>Save PDF</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="editorButton">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	font: 9pt &quot;Fixedsys&quot;;
	border: none;
}
QPushButton:hover{
	color:#D48A21;
}</string>
         </property>
         <property name="text">
          <string>Editor</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="settingButton">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	font: 9pt &quot;Fixedsys&quot;;
	border: none;
}
QPushButton:hover{
	color:#D48A21;
}</string>
         </property>
         <property name="text">
          <string>Settings</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="styleSheet">
       <string notr="true">background-color: #242323;
color : #FFFFFF;</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QTextEdit" name="textEdit">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>750</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 9pt &quot;Fixedsys&quot;;
border:none;</string>
          </property>
          <property name="tabChangesFocus">
           <bool>false</bool>
          </property>
          <property name="documentTitle">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <property name="spacing">
         <number>50</number>
        </property>
        <property name="leftMargin">
         <number>50</number>
        </property>
        <property name="topMargin">
         <number>50</number>
        </property>
        <property name="rightMargin">
         <number>50</number>
        </property>
        <property name="bottomMargin">
         <number>50</number>
        </property>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="tit1">
             <property name="font">
              <font>
               <family>Fixedsys</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Font Size</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBoxFSize">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>250</number>
             </property>
             <property name="value">
              <number>14</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="tit2">
             <property name="font">
              <font>
               <family>Fixedsys</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Font Family</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBoxFamily">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Fixedsys</family>
              </font>
             </property>
             <item>
              <property name="text">
               <string>Segoe UI</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Consolas</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Dejavu Sans</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Noto Sans</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Courier New</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Monaco</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Ubuntu Mono</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Fira Code</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Noto Mono</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Menlo</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_3" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="tit1_2">
             <property name="font">
              <font>
               <family>Fixedsys</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Offsets</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBoxOffsets">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="minimum">
              <number>0</number>
             </property>
             <property name="maximum">
              <number>1000</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_4" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="tit1_3">
             <property name="font">
              <font>
               <family>Fixedsys</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Text Spacing</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBoxTextSpacing">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximum">
              <number>1000</number>
             </property>
             <property name="value">
              <number>20</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
